pkgs/applications/editors/vscode/extensions/rust-lang.rust-analyzer/default.nix

```diff
@@ -39,9 +39,12 @@ let
     inherit releaseTag;
 
     nativeBuildInputs = [
-      jq moreutils esbuild
+      jq
+      moreutils
+      esbuild
       # Required by `keytar`, which is a dependency of `vsce`.
-      pkg-config libsecret
+      pkg-config
+      libsecret
```

Comment:
Maybe you could put these formatting changes in it's own commit, as they are not relevant to the update and clutters the diff?